IntegrationTests/Services/AWSBedrockRuntimeIntegrationTests/BedrockAPIKeyGenerator.swift

```diff
@@ -0,0 +1,106 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import struct AwsCommonRuntimeKit.CommonRuntimeKit
+import struct Foundation.TimeInterval
+import protocol SmithyIdentity.AWSCredentialIdentityResolver
+import protocol SmithyIdentityAPI.IdentityResolver
+import class AWSSDKIdentity.DefaultAWSCredentialIdentityResolverChain
+import class SmithyHTTPAPI.HTTPRequestBuilder
+import struct SmithyHTTPAPI.Headers
+import struct Smithy.URIQueryItem
+import struct Smithy.Attributes
+import enum SmithyHTTPAuthAPI.SigningPropertyKeys
+import class SmithyHTTPAuth.SigV4Signer
```

Comment:
Below is a generator for Bedrock auth tokens.  This is used in integration tests to generate a token for use in the test.

A token is basically a presigned URL with some extra text processing applied to it, as you can see below.

This type is based on the Python token generator package (which despite being a package, is just a few lines of code that use Boto3 to construct the token.)
https://github.com/aws/aws-bedrock-token-generator-python/blob/main/aws_bedrock_token_generator/token_generator.py